odxtools/internalconstr.py

```diff
@@ -0,0 +1,34 @@
+# SPDX-License-Identifier: MIT
+from dataclasses import dataclass
+from typing import List, Optional
+from xml.etree import ElementTree
+
+from .compumethods.limit import Limit
+from .odxtypes import DataType
+from .scaleconstr import ScaleConstr
+
+
+@dataclass
+class InternalConstr:
+    """This class represents a INTERNAL-CONSTR.
```

Comment:
this class now represents both internal and physical constr, so maybe update clasd name and dovs?

Reply:
this is exactly the same as in the XSD, so I think this is okay. (I agree that this should be done differently by the spec...)

Reply:
Class name wise, that's fine due to the spec, but the docs can still be updated, right?

Reply:
true
```suggestion
    """This class represents a INTERNAL-CONSTR objects.

In DOPs, INTERNAL-CONSTR objects can be used to specify validity conditions for internal representations of quantities 
(INTERNAL-CONSTR) and for quantities in the physical domain (PHYS-CONSTR).
```